### ISSUE TITLE
Members in a Space cannot create files despite permission

### ISSUE THREAD

**Issue by pheraph** [humhub/humhub#6120]
Members of a space cannot create new files (uploading, on the other hand, works), although the "Add files" right is permitted for members of the space:

`TypeError: file is undefinedUpload.prototype.handleFileResponse@https://humhub.domain.tld/assets/40215d67/js/humhub.file.js?v=1675242337:270:13
init/ injectedScript:320:19r@https://humhub.domain.tld/assets/38e76184/js/browser/bluebird.min.js?v=1675242336:31:10952`

I am able to reliably reproduce the error on two different humhub instances.

### What steps will reproduce the problem?

Create a space, allow under Security/Permissions that members can create new files and deny that members can manage files. If, on the other hand, file management is allowed, members can create new files. However, the right "Add files" is described in such a way that this alone should be sufficient to create files: "Add files: Allows the user to upload new files and create folders"

### What is the expected result?

Members can create new files without "Manage files" allowed.

### What do you get instead?

Members cannot create new files without "Manage files" allowed.

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   |  1.13.1
| PHP version      | 8.1.16
| Operating system | CentOS Linux

**Comment by ArchBlood** [comment]
Sounds like this is related to the cfile module not the core?

https://github.com/humhub/cfiles

**Comment by pheraph** [comment]
Thanks, that's absolutely possible, I don't have an overview of the inner workings of Humhub yet. If necessary, I think the issue can be moved?

**Comment by luke-** [comment]
Can you reproduce this problem on our demo hosting?

**Comment by pheraph** [comment]
Good hint. I tried to reproduce it there and wasn't able to find the UI element at first. Turns out the issue is with the OnlyOffice-module. After I installed it in the Demo instance I was able to reproduce the problem there, too.

![grafik](https://user-images.githubusercontent.com/58503/220315958-d9c84b5a-034a-4083-a8f0-0a486ec2323f.png)

So I guess I have to place the issue here: https://github.com/ONLYOFFICE/onlyoffice-humhub/issues
